server/src/test/java/org/cloudfoundry/identity/uaa/provider/oauth/ExternalOAuthAuthenticationManagerIT.java

```diff
@@ -263,6 +263,7 @@ void setUp() throws Exception {
                 .setRelyingPartySecret("identitysecret")
                 .setUserInfoUrl(new URL("http://localhost/userinfo"))
                 .setTokenKey(PUBLIC_KEY);
+        config.setAdditionalAuthzParameters(Map.of("token_format", "jwt"));
```

Comment:
Consider adding a test case here that checks whether the additional authz parameters are properly added to the call to the IdP, if possible.